src/Codeception/Module/Db.php

```diff
@@ -534,4 +534,16 @@ public function grabNumRecords($table, array $criteria = [])
     {
         return $this->countInDatabase($table, $criteria);
     }
+    
+    public function updateInDatabase($table, array $data, array $criteria = [])
+    {
+
+        $query = $this->driver->update($table, $data, $criteria);
+        $parameters = array_values($criteria);
```

Comment:
If placeholders are used in SET, $parameters will have to contain $data and $criteria.
```php
$parameters = array_merge(array_values($data), array_values($criteria));
```